.github/ISSUE_TEMPLATE/spec_issue.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Specification Review Issue
+about: Start a request for a specification release
+title: ''
+label: Release Review
+assignees: ''
+---
+
+## Specification issue template
+When creating a specification project lifecycle review, create issues in the  with the content defined as follows.
+- [ ] Specification name and version
+- [ ] Add the label `Release Release`
+- [ ] Naming conventions for artifacts:
+   - [ ] Specification PDF in the form of microprofile-_project_-spec-_version_.pdf
+   - [ ] Specification HTML in the form of microprofile-_project_-spec-_version_.html
+   - _project_ is the microprofile specification short project name (config, health, ...)
+   - _version_ is the two digit x.y version of the specification
+<p>
+
+- [ ] The Nexus Staging links (orgeclipsemicroprofile-NNN where NNN is the staging repository id) which contain all the binaries and relevant documentation:
+   - [ ] [Staging Repo] (add link)
+   - [ ] [Spec PDF] (add link)
+   - [ ] [Spec HTML](add link)
+   - [ ] [Javadoc](add link)
+   - [ ] [API Jar](add link)
+   - [ ] [TCK Jar](add link)
+<p>
+
+- [ ] A link in the Eclipse downloads sections is still available for specs and apidocs:
+   - [ ] https://download.eclipse.org/microprofile/staging/microprofile-_projece_-version/
+   - [ ] An apidocs directory containing the javadoc associated with the API jar.
+<p>
+
+- [ ] [Release Review](https://www.eclipse.org/projects/handbook/#release-review)
+  - [x] Follow [governance process](https://projects.eclipse.org/projects/technology.microprofile/governance) and create/update release record
+    - [Component Release Record](add link to the release record)
+  - [ ] [Email to PMC](mailto:technology-pmc@eclipse.org)
+  - [ ] Start release review by [emailing EMO](mailto:EMO@eclipse-foundation.org) (Track the bug issue created by EMO after the email)
+  - [ ] send an email to [microprofile-wg] (mailto:microprofile-wg@eclipse.org) for voting
+  - [ ] Summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification.
+     - [ ] [Log an issue of Compatible Certification Request] (add link)
+     - [ ][Compatible Implementation TCK Report](add link to the TCK result)
```

Comment:
This can now be removed because it's covered by the CCR.

Reply:
In this case, I can move the CCR to microprofile-config repo. The reason I put here so that I can reference that file.

Reply:
Actually, the template should probably be documented somewhere on our MicroProfile pages...  Not sure where, but we should probably start an "operations guide" explaining these types of steps.

Moving it to the Config repo is fine for now.  We can cut-and-paste to other repos.